Makefile

```diff
@@ -10,7 +10,7 @@ PACKAGES := $$(go list ./... | grep -vE 'vendor')
 build: check test grafana_collector
 
 grafana_collector:
-	$(GO) build -ldflags '$(LDFLAGS)' -o bin/grafana_collector ./metrics/grafana_collector/cmd/grafana_collector/*.go
```

Comment:
just use directory `cmd/grafana_collector/*.go`, remove parent directory `/metrics` and `/metrics/grafana_collector`